### PR TITLE
Fix documentation building

### PR DESCRIPTION
**Pull request by hjmjohnson**

### CODE REVIEW

File: CMake/DownloadDoxygen.cmake [hunk 1]
```diff
@@ -1,3 +1,3 @@
-file( DOWNLOAD http://public.kitware.com/pub/itk/NightlyDoxygen/InsightDoxygenDocHtml.tar.gz
+file( DOWNLOAD https://public.kitware.com/pub/itk/NightlyDoxygen/InsightDoxygenDocHtml.tar.gz
```
Comment (dzenanz) [inline CMake/DownloadDoxygen.cmake]: Was this outdated link somehow responsible for "DoS attack" on Kitware website a few days ago?

File: Superbuild/External-ITK.cmake [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Get and build itk
 
 if(NOT ITK_TAG)
-  set(ITK_TAG "v5.0rc01")
+  set(ITK_TAG "v5.0.0")
```
Comment (dzenanz) [inline Superbuild/External-ITK.cmake]: #103 also updates this, and fixes some other things too.

File: CMake/DownloadDoxygen.cmake (hunk 1 above)
Comment (hjmjohnson) [reply CMake/DownloadDoxygen.cmake]: Perhaps?  I don't know for sure.

File: Superbuild/External-ITK.cmake (hunk 2 above)
Comment (hjmjohnson) [reply Superbuild/External-ITK.cmake]: I know, but #103 is not yet merged. Can you rebase and merge #103 first?

**Pull request merged**